### DISCUSSION TITLE
Apps don't work after re-enabling SIP

### DISCUSSION THREAD

**Discussion by singhalrishi27** [PlayCover/PlayCover#162] [General]
Apps like signal for ipad stops working as soon as I enable sip. SIP is important as it allows you to run legit iOS apps from Mac app store

**Comment by ZhichGaming** [comment]
What boot-args did you put? You can find out by entering `sudo nvram boot-args` in your terminal.

**Comment by singhalrishi27** [comment]
> What boot-args did you put? You can find out by entering `sudo nvram boot-args` in your terminal.

No I didn't put any

**Comment by ZhichGaming** [comment]
What did you need sip disabled for then?

**Comment by singhalrishi27** [comment]
> What did you need sip disabled for then?

To login in apps like YouTube, signal, Subway Surfers

**Comment by ZhichGaming** [comment]
Wait, I reread your question and are you saying that the PlayCover Signal app does not work with SIP enabled? If yes then we unfortunately cannot do anything about it at the moment.

**Comment by ohaiibuzzle** [comment]
Main problem being that without SIP off, we cannot give these applications the set of entitlements that they can only get otherwise by going through Apple's App Store, which are required for some processes like login. There is nothing we can do about it currently.
